src/ajax.js

```diff
@@ -13,6 +13,7 @@ define([
 
 var
 	rhash = /#.*$/,
+	rextraQuestions = /\?+/,
```

Comment:
@jaubourg: i never do some `jsonp` but it's not only these cases:
`index.html?callback=?`
`index.html?var=1&callback=?`
`http://ex.com/?callback=?` and dont find where we have to write two `?`.